.github/workflows/changelog_veriication.yml

```diff
@@ -0,0 +1,25 @@
+name: Changelog Verification
+on:
+  push:
```

Comment:
i think this is missing the `release-` branch name filter.

Also we want to run this on `v3.3.0` tags as well